src/presenter.ts

```diff
@@ -17,29 +19,43 @@ export const presentProfile = (
   subject: NamedNode,
   store: IndexedFormula
 ): ProfilePresentation => {
+  const profile = subject.doc()
   const name = utils.label(subject);
   const imageSrc = findImage(subject);
-  const role = store.anyValue(subject, ns.vcard("role"));
-  const orgName = store.anyValue(subject, ns.vcard("organization-name"));
+  const role = store.anyValue(subject, ns.vcard("role"))
+  const orgName = store.anyValue(subject, ns.vcard("organization-name"), null, profile);
 
-  const address: Node | null = store.any(subject, ns.vcard("hasAddress"));
+  const address: Node | null = store.any(subject, ns.vcard("hasAddress"), null, profile);
   const countryName =
     address != null
-      ? store.anyValue(address as NamedNode, ns.vcard("country-name"))
+      ? store.anyValue(address as NamedNode, ns.vcard("country-name"), null, profile)
       : null;
   const locality =
     address != null
-      ? store.anyValue(address as NamedNode, ns.vcard("locality"))
+      ? store.anyValue(address as NamedNode, ns.vcard("locality"), null, profile)
       : null;
 
   const { backgroundColor, highlightColor } = getColors(subject, store);
 
+  let pronouns = store.anyJS(subject, ns.solid('preferredSubjectPronoun'), null, profile) || ''
```

Comment:
This is all about pronouns and should be extracted to its own function with a respective name. This function could then be unit tested on its own as well